### PR TITLE
add phone numbers for zoom

### PR DESCRIPTION
**Pull request by pjaudiomv**

### CODE REVIEW

**Review by dgershman — APPROVED** [review]

File: index.md [hunk 1]
```diff
@@ -3,7 +3,23 @@ layout: default
 title: BMLT Days 2019
 ---
 
-join info: [https://zoom.us/j/613507415](https://zoom.us/j/613507415)
+*Join info*: [https://zoom.us/j/613507415](https://zoom.us/j/613507415)
+
+*One tap mobile*
+
+[+16465588656,,613507415#](tel:+16465588656,,613507415#) US (New York)
```
Comment (dgershman) [inline index.md:10]: I don't think you can render links with `tel:` in markdown.  At least it doesn't seem to be working.

Comment (pjaudiomv) [reply index.md:10]: it worked here, https://pjaudiomv.github.io/jft/ in my viewer in intellij the link didnt work but it did on github pages

**Review by ChrisMarshallNY — APPROVED** [review]

File: index.md (hunk 1 above)
Comment (dgershman) [reply index.md:10]: It might be just the preview mode on gh.

Comment (dgershman) [reply index.md:10]: Looks good on deploy, thanks!

**Pull request merged**